<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="initStartingCollegesPushButton">
        <property name="text">
         <string>Initialize Starting College</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="initNewCollegesPUshButton">
        <property name="text">
         <string>Initialize New College</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InitializeStartingSouvenirsPushButton">
        <property name="text">
         <string>Initialize Starting Souvenirs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="InitializeNewSouvenirsPushButton">
        <property name="text">
         <string>Initialize New Souvenirs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteTablePushButton">
        <property name="text">
         <string>Delete Tables</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>181</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7"/>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableView" name="distanceTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>College</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="collegeLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Ending College</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="endingCollegeLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Distance (km)</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="kilometerLineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="addCollegeButton">
            <property name="text">
             <string>Add College</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTableView" name="souvenirTableView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>College</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="collegeLineEdit2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Souvenir</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="souvenirLineEdit"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Cost</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="costLineEdit"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="addSouvenirButton">
            <property name="text">
             <string>Add Souvenir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editSouvenirButton">
            <property name="text">
             <string>Edit Souvenir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteSouvenir">
            <property name="text">
             <string>Delete Souvenir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSales_Report">
   <property name="text">
    <string>Sales Report</string>
   </property>
  </action>
  <action name="actionPurchases_Char">
   <property name="text">
    <string>Purchases Char</string>
   </property>
  </action>
  <action name="actionPurchases_Chart">
   <property name="text">
    <string>Purchases Chart</string>
   </property>
  </action>
  <action name="actionMember_Purchases">
   <property name="text">
    <string>Member Purchases</string>
   </property>
  </action>
  <action name="actionManage_Inventory">
   <property name="text">
    <string>Manage Inventory</string>
   </property>
  </action>
  <action name="actionItem_List">
   <property name="text">
    <string>Inventory List</string>
   </property>
  </action>
  <action name="actionItem_Search">
   <property name="text">
    <string>Item Search</string>
   </property>
  </action>
  <action name="actionManage_Members">
   <property name="text">
    <string>Manage Members</string>
   </property>
  </action>
  <action name="actionMember_Search">
   <property name="text">
    <string>Member Search</string>
   </property>
  </action>
  <action name="actionUpgrade_Downgrade_List">
   <property name="text">
    <string>Upgrade/Downgrade List</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
